base/applications/notepad/settings.c

```diff
@@ -170,7 +167,7 @@ void NOTEPAD_LoadSettingsFromRegistry(void)
     else
     {
         /* If no settings are found in the registry, then use default values */
-        Globals.bShowStatusBar = FALSE;
+        Globals.bShowStatusBar = TRUE;
```

Comment:
Is it a better default? What does Windows appear to default to?

Reply:
Wouldn't that new default also require us to ensure, that wordwrap would be turned off? (based on your most recent findings)

Reply:
![VirtualBox_Windows Server 2003_15_02_2023_10_13_54](https://user-images.githubusercontent.com/2107452/218899688-95df8ceb-de22-4b0c-8f7c-abc0c9d38663.png)
Win2k3 default is `FALSE`.
Win10 default is `TRUE`.

Reply:
![2k3](https://user-images.githubusercontent.com/2107452/218909676-8003d6bb-73f4-422d-a59a-373a39f9d88c.png)
Win2k3 `bWrapLongLines` default is `FALSE`.
Win10 `bWrapLongLines` default is `FALSE`.